src/GameLoop.tsx

```diff
@@ -554,3 +583,21 @@ export function GameLoop() {
         </div>
     );
 }
+
+export const CountdownOverlay = React.memo(
+    ({ isVisible, countdownNum }: { isVisible: boolean, countdownNum: number }) => {
+        const divStyle = {
+            visibility: isVisible ? "visible" as const : "hidden" as const,
+            position: 'absolute',
+            top: '35%',
+            left: '50%',
+            transform: 'translate(-50%, -50%)',
+            zIndex: 2,
+            color: 'red',
+            fontSize: '200%',
+        };
+        return <div style={divStyle}>
+            {countdownNum}
```

Comment:
I'm still not sure what `countdownNum` is.

Reply:
`countdownNum` is an integer in the range `countdownTotalSteps` to 0. It decreases with each second. Its purpose is to be displayed to the player during the countdown sequence

Reply:
Can it be`countdownSecs` then?

Reply:
Done